### ISSUE TITLE
Sign tCRIL up as a Django Corporate Sponsor

### ISSUE THREAD

**Issue by ormsbee** [openedx/tcril-engineering#260]
We'd go for [Bronze membership](https://www.djangoproject.com/foundation/corporate-membership/) for now (10 people), $2,000.

- [ ] Submit application form.
- [ ] ...?

It looks like they field these applications after their monthly meetings, so we might not get a response for a while.

**Comment by ormsbee** [comment]
@e0d: I forwarded the invoice to you last week. Assigning this ticket to you. Please assign back if there's anything you'd like me to do.

**Comment by e0d** [comment]
@ormsbee this is good to close, correct?

**Comment by ormsbee** [comment]
Yup, we're on [the sponsors page](https://www.djangoproject.com/foundation/corporate-members/)! Thank you!
